.github/ISSUE_TEMPLATE.md

```diff
@@ -2,15 +2,23 @@
 <!-- If you report a security issue, please refrain from filling an issue and refer to SECURITY.md for the disclosure procedure. -->
 <!-- If you report a bug, please fill the form -->
 
-* **PeerTube version or URL:**
-* **Browser name/version:**
-* **NodeJS version:**
+**What happened?**
 
+**What do you expect to happen instead?**
 
-* **What is the expected behaviour?**
-* **What do you see instead?**
+**Steps to reproduce:**
 
+1.
 
-* **Browser console log if useful:**
-* **Server log if useful (journalctl or /var/www/peertube/storage/logs):**
+2.
 
+3.
+
+
+**Additional information**
+* PeerTube version or URL:
+* Browser name/version:
+* NodeJS version:
+
+* Link to browser console log if useful:
+* Link to server log if useful (journalctl or /var/www/peertube/storage/logs):
```

Comment:
Maybe if /var/www/peertube/storage/logs finish with a / that will be more clear (for indicate that the logs are in the folder, not a single file).

Reply:
:+1:

Reply:
Makes sense, fixed :)